client/src/components/join/Join.tsx

```diff
@@ -0,0 +1,52 @@
+import { useState } from "react";
+import { IJoinFormData } from "../../interfaces/JoinFormData";
+import { useHistory } from "react-router-dom";
+import "./Join.css";
+import routes_ from "../../constants/Route";
+
+function getSessionId(path: string) {
+  const pathParts = path.split("/");
+  return pathParts[pathParts.length - 1];
+}
+
+function Join() {
```

Comment:
Just curious if we can reuse the component from the homepage with the session_id filled in to the input text field

Reply:
Nice idea